Examples/CaseStudies/SwiftUICaseStudies/02-Effects-WebSocket.swift

```diff
@@ -135,41 +136,48 @@ struct WebSocketView: View {
 
   var body: some View {
     WithViewStore(self.store, observe: { $0 }) { viewStore in
-      VStack(alignment: .leading) {
-        AboutView(readMe: readMe)
-          .padding(.bottom)
-
-        HStack {
-          TextField(
-            "Message to send",
-            text: viewStore.binding(
-              get: \.messageToSend, send: WebSocket.Action.messageToSendChanged)
-          )
-
-          Button(
-            viewStore.connectivityState == .connected
+      Form {
+        Section {
+          AboutView(readMe: readMe)
+        }
+
+        Section {
+          VStack(alignment: .leading) {
+            Button(
+              viewStore.connectivityState == .connected
               ? "Disconnect"
               : viewStore.connectivityState == .disconnected
                 ? "Connect"
                 : "Connecting..."
-          ) {
-            viewStore.send(.connectButtonTapped)
+            ) {
+              viewStore.send(.connectButtonTapped)
+            }
+            .buttonStyle(.bordered)
+            .tint(viewStore.connectivityState == .connected ? .red : .green)
+
+            HStack {
+              TextField(
+                "Type message here",
+                text: viewStore.binding(
+                  get: \.messageToSend, send: WebSocket.Action.messageToSendChanged)
+              )
+              .textFieldStyle(.roundedBorder)
+
+              Button("Send") {
+                viewStore.send(.sendButtonTapped)
+              }
+              .buttonStyle(.borderless)
+            }
           }
         }
 
-        Button("Send message") {
-          viewStore.send(.sendButtonTapped)
+        Section(
+          header: Text("Received messages")) {
+            Text("Status: \(viewStore.connectivityState.rawValue)")
+              .foregroundStyle(.secondary)
+            Text(viewStore.receivedMessages.reversed().joined(separator: "\n"))
         }
```

Comment:
This initializer is soft-deprecated in favor of the one that position `header` and `footer` after `content`.
```suggestion
        Section {
          Text("Status: \(viewStore.connectivityState.rawValue)")
            .foregroundStyle(.secondary)
          Text(viewStore.receivedMessages.reversed().joined(separator: "\n"))
        } header: { 
          Text("Received messages") 
        }
```

Reply:
Thanks! @tgrapperon 👍🏻